q2_feature_table/tests/test_merge.py

```diff
@@ -302,5 +303,12 @@ def test_merge_taxa(self):
         pdt.assert_frame_equal(obs, exp)
 
 
+class TestUsageExample(TestPluginBase):
+    package = 'q2_feature_table.tests'
+
+    def test_usage_example_ft_merge(self):
+        self.execute_examples()
+
+
```

Comment:
Please move this into a new test file (`test_examples.py`) - the machinery used here executes all examples - so it isn't really specific to testing merging, which will make this location a bit confusing down the line.